homeassistant/components/media_player/kodi.py

```diff
@@ -364,6 +378,36 @@ def ws_loop_wrapper():
         self.hass.loop.create_task(ws_loop_wrapper())
 
     @asyncio.coroutine
+    def async_added_to_hass(self):
+        """Script creation for the turn on/off config options."""
+        if self._turn_on_action is not None:
+            self._turn_on_action = script.Script(
+                self.hass, self._turn_on_action, "Kodi Turn ON action script")
+        if self._turn_off_action is not None:
+            # Deprecation check
+            if isinstance(self._turn_off_action, str):
```

Comment:
Agree with @azogue . Backwards compatibility wouldn't be a requirement to make this change, but if it's already written I'd prefer to keep it to make the transition smooth for everyone.